core-js/src/main/javascript/cdf/components/ccc/vizApi.conf.js

```diff
@@ -0,0 +1,646 @@
+/*!
+ * Copyright 2016 - 2017 Pentaho Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+define([
+  "pentaho/visual/color/paletteRegistry"
+], function(visualPaletteRegistry) {
+
+  "use strict";
+
+  var cdfFont = "10px OpenSansRegular";
+  var maxHorizontalTextWidth = 117;
+  var pvc = null;
+  var numberFormatCache = {};
+
+  var numberStyle = {
+    group: " ",
+    abbreviations:    ['k', 'M', 'G', 'T', 'P', 'E', 'Z', 'Y'],
+    subAbbreviations: ['m', 'µ', 'n', 'p', 'f', 'a', 'z', 'y']
+  };
+
+  configureVizAPIPalette();
+
+  /* eslint camelcase: "off" */
+  /* eslint require-jsdoc: "off" */
+  /* eslint brace-style:0 */
+  /* eslint key-spacing:0 */
+  return {
+    rules: [
+
+      // CCC Abstract
+      {
+        select: {
+          priority: -1,
+          application: "pentaho-cdf",
+          type: "pentaho/ccc/visual/abstract"
+        },
+        apply: {
+          extension: {
+
+            // Chart
+            format: {
+              number: {
+                style: numberStyle
+              }
+            },
+
+            errorMessage_visible: false,
+            noDataMessage_visible: false,
+            invalidDataMessage_visible: false,
+
+            plotFrameVisible: false,
+            plotSizeMin: {width: 600, height: 300},
+
+            clickable:  false,
+            selectable: false,
+            hoverable:  true,
```

Comment:
`clickable` should be removed.
Not sure if `selectable` should be set to `false` — only if some of the styling changes we did don't work well with selection...